app/topics_handler.go

```diff
@@ -107,6 +107,10 @@ func (th *TopicsHandler) requestTopicReputers(ctx sdk.Context, topic emissionsty
 			ctx.Logger().Error("Error getting latest inferences at block: ", nonceCopy.ReputerNonce.BlockHeight, ", error: ", err)
 			continue
 		}
+		if len(reputerValueBundle.InfererValues) == 0 {
```

Comment:
if reputerValueBundle is actually nil, this will panic :P 
that needs to be checked too

if reputerValueBundle == nil || len(reputerValueBundle.InfererValues) == 0 {

Reply:
okay, I didn't add it because it doesn't return nil in any case. Anyway, it's good to add this one.